bluebutton-data-pipeline-fhir-load/src/test/java/gov/hhs/cms/bluebutton/datapipeline/fhir/transform/DataTransformerTest.java

```diff
@@ -202,12 +205,8 @@ public void transformInpatientClaim() throws FHIRException {
 		ItemsComponent eobSoleItem = eob.getItem().get(0);
 		Assert.assertEquals(Date.valueOf(sourceClaim.getDateAdmission()), eobSoleItem.getServicedPeriod().getStart());
 		Assert.assertEquals(Date.valueOf(sourceClaim.getDateDischarge()), eobSoleItem.getServicedPeriod().getEnd());
-		Assert.assertEquals(
-				eob.getProvider()
-						.getReference(),
-				bundle.getFhirResources().stream()
-						.filter(r -> r instanceof Practitioner).map(
-								r -> (Practitioner) r)
+		Assert.assertEquals(eob.getProvider().getReference(),
+				bundle.getFhirResources().stream().filter(r -> r instanceof Practitioner).map(r -> (Practitioner) r)
 						.filter(p -> p.getIdentifier().stream()
```

Comment:
@karlmdavis I did go through those instructions, but in Eclipse the "Format all lines" option is the one that is defaulted, instead of "Format edited lines" which it sounds like would be the preferred option?